File: FrozenApp/src/main/java/com/github/lakeshire/frozen/ListEntity.java
package com.github.lakeshire.frozen;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by liuhan on 17/9/1.
 */

public class ListEntity<T> {
    public int start;
    public int count;
    public int total;
    public String title;

    @SerializedName("subjects")
    public List<T> list;
}

File: FrozenApp/src/main/java/com/github/lakeshire/frozen/ListEntity.kt
package com.github.lakeshire.frozen

import com.google.gson.annotations.SerializedName

class ListEntity<T> {
    var start: Int = 0
    var count: Int = 0
    var total: Int = 0
    var title: String? = null

    @SerializedName("subjects")
    var list: List<T>? = null
}
